src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -959,6 +960,14 @@ public static partial class JsonMetadataServices
         public static System.Text.Json.Serialization.JsonConverter<T> GetEnumConverter<T>(System.Text.Json.JsonSerializerOptions options) where T : struct { throw null; }
         public static System.Text.Json.Serialization.JsonConverter<T?> GetNullableConverter<T>(System.Text.Json.Serialization.Metadata.JsonTypeInfo<T> underlyingTypeInfo) where T : struct { throw null; }
     }
+    public struct JsonParameterClrInfo
```

Comment:
Do we use the term `Clr` already in public API? I thought we wanted to stay away from that term in the API review previously.

Reply:
If "Clr" here is referring to the runtime, yes, we shouldn't use it.

Did all of this already go through API review?

Reply:
I used "Clr" here because we have an internal type called `JsonParameterInfo`. I'll rename this to `JsonParameterInfo` & the internal type to something like `JsonParameterInfoCore`.

It hasn't gone through review yet but I've added a proposal here which should be reviewed soon - https://github.com/dotnet/runtime/issues/45448#issuecomment-887926294.